allensdk/ephys/ephys_features.py

```diff
@@ -466,8 +466,9 @@ def find_widths(v, t, spike_indexes, peak_indexes, trough_indexes, clipped=None)
 
     # Some spikes in burst may have deep trough but short height, so can't use same
     # definition for width
-    width_levels[width_levels < v[spike_indexes]] = \
-        thresh_to_peak_levels[width_levels < v[spike_indexes]]
+    with warnings.catch_warnings():
```

Comment:
Can you instead use a
```
with np.errstate(invalid='ignore'):
```
context here. It feels more targeted. I'm uneasy catching and ignoring **all** `RuntimeWarning`s